### PR TITLE
LB-1417: Add recording artists to release group cache

### PR DESCRIPTION
**Pull request by amCap1712**

### CODE REVIEW

File: frontend/js/src/entity-pages/AlbumPage.tsx [hunk 1]
```diff
@@ -164,13 +168,16 @@ export default function AlbumPage(props: AlbumPageProps): JSX.Element {
...
               merge(track, {
-                artist_name: artistName,
-                artist_mbids:
-                  artist?.artists?.map((ar) => ar.artist_mbid) ?? [],
+                artist_name: track.artists
+                  .map(
+                    (track_artist) =>
+                      `${track_artist.artist_credit_name}${track_artist.join_phrase}`
+                  )
+                  .join(""),
```
Comment (MonkeyDo) [inline frontend/js/src/entity-pages/AlbumPage.tsx:176]: Let's do this for now; in the future I think we can refactor the [`getArtistName` utility](https://github.com/metabrainz/listenbrainz-server/blob/master/frontend/js/src/utils/utils.tsx#L231) to accept more input formats.

We probably want to keep the artist_mbids in there, as they can be used for other purposes down the line such as CB reviews

File: frontend/js/src/entity-pages/AlbumPage.tsx [hunk 2]
```diff
@@ -164,13 +168,16 @@ export default function AlbumPage(props: AlbumPageProps): JSX.Element {
...
                 caa_id: Number(caa_id) || undefined,
                 caa_release_mbid,
                 recording_name: track.name,
-                release_mbid: caa_release_mbid ?? "",
+                release_mbid: recordings_release_mbid ?? "",
```
Comment (MonkeyDo) [inline frontend/js/src/entity-pages/AlbumPage.tsx:180]: Would this make sense, just to ensure a cover art fallback?
```suggestion
                release_mbid: recordings_release_mbid ?? caa_release_mbid ?? "",
```

**Review by MonkeyDo — COMMENTED** [review]
Good improvement

**Comment by MonkeyDo** [comment]
Tested live on test.LB, working marvelously ! 💯

**Review by mayhem — APPROVED** [review]
Yeah, good improvement!

File: frontend/js/src/entity-pages/AlbumPage.tsx (hunk 1 above)
Comment (amCap1712) [reply frontend/js/src/entity-pages/AlbumPage.tsx:176]: Sounds good.

**Pull request merged**